### ISSUE TITLE
GBC: Tigger's Hunni Safari, major graphical glitches

### ISSUE THREAD

**Issue by matthargett** [SmokeMonsterPacks/Nt-Mini-Noir-Jailbreak#13]
On start menu after the intro animations looks okay, persists into the game.
![image](https://user-images.githubusercontent.com/1550766/100272489-113b1900-2f10-11eb-8a43-59118f161622.png)

**Comment by frederic-mahe** [comment]
Possible hash values for these ROMs are:
```
0738cbd9348166ea844bfdf76a24ea5e6bd9aebe8411e192c32b9ec40b978f15	Pooh and Tigger's Hunny Safari (USA).gbc	934918da6249723760a356748bf240f57262bd77	5f872189d58f841b5672d4cb11ec3bbb	622690da
c6de43ecf5c6fe1722b5b6d6ac375cba14c37db04d29455c295cfad4fc55b864	Pooh and Tigger's Hunny Safari (Europe) (En,Fr,De,Es,It,Nl).gbc	4141887f0b152f8592b905355e11cf43147128c2	7a58b84e406bd4df6b3c3b8807493aa8	8505d139
```
@matthargett could you please confirm that you have the same ROMs?

**Comment by matthargett** [comment]
confirmed, it's the hash value for the USA one. dumped directly from my physical cartridge.

**Comment by matthargett** [comment]
Fixed for me in 6.5 update.
